endoreg_db/models/media/video/video_file_anonymize.py

```
@@ -285,7 +419,9 @@ def _anonymize(video: "VideoFile", delete_original_raw: bool = True) -> bool:
 
             transaction.on_commit(
                 lambda: _cleanup_raw_assets(
-                    video_uuid=video.uuid, raw_file_path=original_raw_file_path_to_delete, raw_frame_dir=original_raw_frame_dir_to_delete
+                    video_content_hash=video.video_hash,
+                    raw_file_path=original_raw_file_path_to_delete,
+                    raw_frame_dir=original_raw_frame_dir_to_delete,
                 )
```

Pylint (4.0.4)

[error] Unexpected keyword argument 'video_content_hash' in function call

(E1123)